### ISSUE TITLE
Output HTTP status code as well when logging KeycloakApiException

### ISSUE THREAD

**Issue by IngoStrauch2020** [mayope/keycloakmigration#55]
It's maybe just a nice to have, but when e.g. the admin user executing a migration does not have the "manage-authorization" role and tries to use the "addFlow" migration, the error message is just

```
Error occurred while migrating: 56_add_new_realm_idp_dev
de.klg71.keycloakmigration.keycloakapi.KeycloakApiException: {"error":"unknown_error"}
```

Calling the REST API with some other tool would show that the HTTP status code is 403 Forbidden, which is the relevant information here.

Maybe the log could be changed like this
```
Error occurred while migrating: 56_add_new_realm_idp_dev
de.klg71.keycloakmigration.keycloakapi.KeycloakApiException: code=403, response={"error":"unknown_error"}
```

**Comment by IngoStrauch2020** [comment]
I could imagine two easy ways to do it.

a) Change the one relevant code line to pass the final message

```
fun Response.extractLocationUUID(): UUID {
    if (!isSuccessful()) {
        throw KeycloakApiException("code=${this.status()}, response=${this.body().asReader().readText()})
    }
    ...
}
```

b) add a second constructor to the KeycloakApiException to centralize the message generation for future use

```
class KeycloakApiException : RuntimeException {
    constructor(message: String): super(message)
    constructor(code: Int, message: String): super("code=$code, response=$message")
}
```
plus
```
fun Response.extractLocationUUID(): UUID {
    if (!isSuccessful()) {
        throw KeycloakApiException(this.status(), this.body().asReader().readText())
    }
    ...
}
```

**Comment by klg71** [comment]
Hey Ingo,
sry I forgot about this issue and only implemented it halfway through. I will get to it this week :)

**Comment by klg71** [comment]
Released with version 0.2.46 :) Maybe you could verify this works

**Comment by IngoStrauch2020** [comment]
Sorry, better late than never. It works, thanks!
